lte/gateway/c/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -460,6 +460,7 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
 
   put_mme_nas_state();
   put_mme_ue_state(mme_app_desc_p, imsi64);
+  put_mme_ueip_imsi_map();
```

Comment:
we should not use this pattern at all. let's directly write to store when actually it is written.

Reply:
Corrected code such that when in-memory map is modified, it's written DB at the same time